### PR TITLE
SonarQube bugfixes

### PR DESCRIPTION
**Pull request by hdenes97**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -9,6 +9,8 @@
 */
 public class TorpedoStore {
 
+  private static Random generator = new Random();
```
Comment (FP5i) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Use SecureRandom

**Review by FP5i — CHANGES_REQUESTED** [review]
Well done, I have only one change request.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (hdenes97) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: No

**Comment by FP5i** [comment]
OK

**Pull request merged**